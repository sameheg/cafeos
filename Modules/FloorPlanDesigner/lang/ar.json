{
  "title": "مصمم مخطط الطابق",
  "addTable": "إضافة طاولة",
  "addChair": "إضافة كرسي",
  "save": "حفظ التخطيط"
}
