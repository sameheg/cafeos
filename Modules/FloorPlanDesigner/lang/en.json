{
  "title": "Floor Plan Designer",
  "addTable": "Add table",
  "addChair": "Add chair",
  "save": "Save Layout"
}
