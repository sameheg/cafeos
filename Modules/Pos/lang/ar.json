{
  "enabled": "تم تفعيل وحدة نقاط البيع",
  "created": "تم إنشاء عنصر القائمة",
  "updated": "تم تحديث عنصر القائمة",
  "deleted": "تم حذف عنصر القائمة",
  "moved": "تم نقل الطلب",
  "split": "تم تقسيم الفاتورة بنجاح"
}
