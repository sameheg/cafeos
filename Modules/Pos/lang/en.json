{
  "enabled": "POS module is enabled",
  "created": "Menu item created",
  "updated": "Menu item updated",
  "deleted": "Menu item deleted",
  "moved": "Order moved",
  "split": "Bill split successfully"
}
