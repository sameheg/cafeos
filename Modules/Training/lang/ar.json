{
    "training_material": "مواد التدريب",
    "training_completion": "إكمال التدريب"
}
