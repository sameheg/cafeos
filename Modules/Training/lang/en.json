{
    "training_material": "Training Material",
    "training_completion": "Training Completion"
}
