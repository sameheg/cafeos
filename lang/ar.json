{
  "dashboard.title": "لوحة تحكم نقاط البيع",
  "dashboard.stock": "المخزون",
  "welcome.title": "كافيه أو إس",
  "pos.enabled": "تم تفعيل وحدة نقاط البيع",
  "pos.created": "تم إنشاء عنصر القائمة",
  "pos.updated": "تم تحديث عنصر القائمة",
  "pos.deleted": "تم حذف عنصر القائمة",
  "pos.moved": "تم نقل الطلب",
  "pos.split": "تم تقسيم الفاتورة بنجاح",
  "floorPlanDesigner.title": "مصمم مخطط الطابق",
  "floorPlanDesigner.addTable": "إضافة طاولة",
  "floorPlanDesigner.addChair": "إضافة كرسي",
  "floorPlanDesigner.save": "حفظ التخطيط"
}
