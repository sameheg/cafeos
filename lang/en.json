{
  "dashboard.title": "POS Dashboard",
  "dashboard.stock": "Stock",
  "welcome.title": "CafeOS"
}
