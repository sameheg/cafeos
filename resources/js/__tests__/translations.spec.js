import en from '../../../lang/en.json';
import ar from '../../../lang/ar.json';

describe('translation keys', () => {
  const enKeys = Object.keys(en).sort();
  const arKeys = Object.keys(ar).sort();

  it('english keys match snapshot', () => {
    expect(enKeys).toMatchSnapshot();
  });

  it('arabic contains all english keys', () => {
    expect(arKeys).toEqual(enKeys);
  });
});
