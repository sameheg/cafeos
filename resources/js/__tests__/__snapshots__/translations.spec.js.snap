// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`translation keys > english keys match snapshot 1`] = `
[
  "dashboard.stock",
  "dashboard.title",
  "welcome.title",
]
`;
